cmd/token.go

```diff
@@ -138,7 +138,7 @@ The OIDC token includes claims regarding the GCE VM, which is verified by Attest
 			Challenge:      challenge,
 			GcpCredentials: principalTokens,
 			Attestation:    attestation,
-			TokenOptions:   verifier.TokenOptions{CustomAudience: audience, TokenType: "OIDC"},
+			TokenOptions:   verifier.TokenOptions{CustomAudience: audience, CustomNonce: customNonce, TokenType: "OIDC"},
```

Comment:
Does this mean that default token will have this custom nonce every time? What is this used for?

Reply:
The default value for `customNonce` is nil. Thus, the default token would not contain custom nonce. This is used for on-demand attestation for CVMs for customer to build upon.

Reply:
Ah yes sorry, totally missed the directory this was in.